src/commands/BootstrapCommand.js

```diff
@@ -92,18 +92,40 @@ export default class BootstrapCommand extends Command {
   bootstrapPackages(callback) {
     this.logger.info("", `Bootstrapping ${this.filteredPackages.length} packages`);
 
-    async.series([
-      // preinstall bootstrapped packages
-      (cb) => this.preinstallPackages(cb),
-      // install external dependencies
-      (cb) => this.installExternalDependencies(cb),
-      // symlink packages and their binaries
-      (cb) => this.symlinkPackages(cb),
-      // postinstall bootstrapped packages
-      (cb) => this.postinstallPackages(cb),
-      // prepublish bootstrapped packages
-      (cb) => this.prepublishPackages(cb)
-    ], callback);
+    const { yarnWorkspaces } = this.options;
+
+    if (yarnWorkspaces && (yarnWorkspaces.indexOf("bootstrap") >= 0)) {
+      this.installRootPackageOnly(callback);
```

Comment:
Yep, workspaces feature in Yarn creates symlinks in node_modules.
It also treats all workspaces as one installation, so it would hoist symlinks to the root node_modules unless there is a conflict, e.g. jest@next might be using jest-cli@stable in the root.